meilisearch/task.py

```diff
@@ -6,7 +7,7 @@
 from meilisearch.config import Config
 from meilisearch.errors import MeiliSearchTimeoutError
 
-def get_tasks(config: Config, index_id: Optional[str] = None) -> Dict[str, List[Dict[str, Any]]]:
+def get_tasks(config: Config, index_ids: Optional[List[str]] = None, resource_query: Optional[Dict[str, Any]] = None) -> Dict[str, List[Dict[str, Any]]]:
```

Comment:
`index_ids` should be inside `resource_query`

Reply:
This function will never be called by users. This parameter is only used by the `index.get_task()` function. So I removed the modification made for this comment https://github.com/meilisearch/meilisearch-python/pull/476#discussion_r899028374 because this case will never happen.
I rearranged the function in case the user would add other indexes

Reply:
The user could do 
`client.get_tasks({ 'indexUid': ['movies', 'test'] })`

and behind they are using the same method:
```
def get_tasks(config: Config, index_ids: Optional[List[str]] = None, resource_query: Optional[Dict[str, Any]] = None)
```

So `index_ids` is not required as argument since it is already possible with `resource_query`